mlflow/models/evaluation/base.py

```diff
@@ -1769,6 +1813,16 @@ def pred_sample(eval_df, _builtin_metrics, _artifacts_dir):
     from mlflow.pyfunc import PyFuncModel, _load_model_or_server, _ServedPyFuncModel
     from mlflow.utils import env_manager as _EnvManager
 
+    # Inference params are currently only supported for passing a deployment endpoint as the model.
+    # TODO: We should support inference_params for other model types
+    is_model_endpoint_url = isinstance(model, str) and model.startswith("endpoints:/")
```

Comment:
I think `startswith` is somewhat brittle, e.g. 3 slashes is technically also valid. Can we use urlparse and check the scheme?